gpkit/nomials/nomial_core.py

```diff
@@ -61,7 +62,7 @@ def latex(self, excluded=None):
         if "units" in excluded:
             return " + ".join(sorted(mstrs))
 
-        units = unitstr(self.units, r"\mathrm{~\left[ %s \right]}", "L~")
+        units = self.unitstr(r"\mathrm{~\left[ %s \right]}", "L~")
         units_tf = units.replace("frac", "tfrac").replace(r"\cdot", r"\cdot ")
```

Comment:
seems like this replacement stuff belongs in `unitstr`? Does it really only apply to a `Nomial` and no other GPkit objects?